### DISCUSSION TITLE
IQ Tree Web Server ERROR: Not enough memory!

### DISCUSSION THREAD

**Discussion by ranjan-tiwari** [iqtree/iqtree2#205] [Q&A] [answered]
Greetings,

While using the IQ-Tree Webserver to construct a maximum likelihood tree with 10,000 ultrafast bootstraps, I encountered the following error in the run log: "ERROR: Not enough memory!"

Can anyone tell me if running 10,000 bootstraps is feasible with this server, or does this error always occur?

Thanks,

**Comment by bqminh** [comment, marked as answer]
The error message means that your dataset is too large for the web server and thus can't run it. You can download and run the command line version. See this quick start: http://www.iqtree.org/doc/Quickstart
